tests/spread/plugins/craft-parts/kernel/snapcraft.yaml

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021 Canonical Ltd
+name: pc-kernel
+summary: Generic Ubuntu pc-kernel
+description: The generic Ubuntu pc-kernel
+
+grade: stable
+build-base: core22
+confinement: strict
+type: kernel
+adopt-info: kernel
+
+architectures:
+  - build-on: [amd64]
+    build-for: amd64
+#  - build-on: [amd64, arm64]
```

Comment:
If there is a reason to keep this dead code, can you add a comment?

Reply:
The kernel is specific to amd64, will remove the commented lines.